bayesian/utils.py

```diff
@@ -339,12 +340,82 @@ def build_nested_schema_dict(schema_dict):
     return result
 
 
+def get_next_component_from_graph(ecosystem, user_id, company):
+    tag_count = 2
```

Comment:
@ravsa this tag count should be provided as a configurable in `default_config.py` and it should be made available through an environment variable.

You can make an entry for a new environment variable in openshift/template.yaml.